app/assets/javascripts/markdown.js

```diff
@@ -0,0 +1,45 @@
+$( document ).ready(function() {
```

Comment:
@lisbethmarianne, do you wanna delete the spaces before and after `document`? :)
Feel better soon btw!